README.md

```diff
@@ -138,6 +139,25 @@ schema.cast({
 // => { name: 'jimmy', age: 24, createdOn: Date }
 ```
 
+### Using a custom locale dictionary
+```js
+import { setLocale } from 'yup/lib/locale'
+
+setLocale({
+  string: {
+    email: 'Invalid',
+  },
+})
+
+// Now use Yup schemas AFTER you defined your custom dicionary
+```
+Why do you have to set the locales before you use Yup at all?
```

Comment:
I was think more about what this does generally, not why it needs be first. something like: 

"Allows you to customize the default messages used by Yup, when no message is provided with a validation test."